apps/aechannel/src/aesc_force_progress_tx.erl

```diff
@@ -391,21 +358,14 @@ state_hash(#channel_force_progress_tx{state_hash = StateHash}) ->
     StateHash.
 
 -spec version(tx()) -> non_neg_integer().
-version(#channel_force_progress_tx{block_hash = BlockHash}) ->
-    case BlockHash =:= aesc_pinned_block:no_hash() of
-        true ->
-            ?INITIAL_VSN;
-        false ->
-            ?PINNED_BLOCK_VSN
-    end.
+version(_) ->
+    ?LATEST_VSN.
```

Comment:
It seems to me we rather want to keep INITIAL_VSN i.e. 1, and optionally skip 2 in future?

Reply:
I need to understand better if this is in serialization or DB though.

Reply:
As I understand from https://github.com/aeternity/protocol/pull/341/files
using 3 is unneeded. A simple revert to 1 is ok?

Also - could this version 2 anywhere got to the DB for supported runtime i.e. Minerva? I expect no.